platform-api/che-core-api-infrastructure-local/src/main/java/org/eclipse/che/api/local/LocalWorkspaceDaoImpl.java

```diff
@@ -55,7 +54,9 @@
 
     @Inject
     public LocalWorkspaceDaoImpl(LocalStorageFactory factory) throws IOException {
-        final Map<Class<?>, Object> adapters = ImmutableMap.of(Recipe.class, new RecipeTypeAdapter());
+        final Map<Class<?>, Object> adapters = new HashMap<>();
```

Comment:
It is okay. Just note that `ImmutableMap.of` may be used with as many values as you want.
E.g 
```java
ImmutableMap.of(Recipe.class, new RecipeTypeAdapter(), 
                ModuleConfig.class, new ModuleConfigAdapter());
```